content/questions/async_await/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Async/Await
+tags:
+  - es6
+  - javascript
+  - promise
+order: 39
+date: Mon Oct 07 2019 09:40:20 GMT-0700 (Mountain Standard Time)
+answers:
+  - 'first!'
+  - 'done! // correct'
+  - 'JavaScript error'
+  - 'Something else'
+---
+
+Consider the following async function and it's output, what will be displayed to the console when
+calling the `f()` function?
+
+```javascript
+async function f() {
+  let result = 'first!';
+  let promise = new Promise((resolve, reject) => {
+    setTimeout(() => resolve('done!'), 1000);
+  });
+
+  let result = await promise;
```

Comment:
This will error for re-declaring result, right? should this just be `result = await promise`?

Reply:
Duh! I'll get that fixed 👍